src/components/CashoutModal.tsx

```diff
@@ -27,20 +29,31 @@ export default function DepositModal() {
 
   const handleCashout = () => {
     if (peerId) {
+        setLoadingCashout(true)
         beeDebugApi.chequebook.peerCashout(peerId)
         .then(res => {
             setOpen(false);
-            handleToast(`Successfully cashed out cheque. Transaction ${res.data.transactionHash}`)
+            handleToast(<span>Successfully cashed out cheque. Transaction 
+              <EthereumAddress
+              hideBlockie
```

Comment:
This looks like a typo

Reply:
Yeah I wondered as well but it is these ethereum identicons https://www.npmjs.com/package/ethereum-blockies